api/endpoint-tests/activities/goals/put.js

```diff
@@ -98,19 +98,23 @@ tap.test(
               200,
               'gives a 200 status code'
             );
+
+            body.goals.forEach(goal => goal.objectives.sort());
```

Comment:
It's really obnoxious having to sort all these arrays to make the tests pass, but maybe these are actual bugs - see the question in Slack about whether these should be returned in a consistent order!